libraries/node-core-library/src/PackageJsonLookup.ts

```diff
@@ -224,7 +224,7 @@ export class PackageJsonLookup {
         packageJson.peerDependencies = loadedPackageJson.peerDependencies;
         packageJson.private = loadedPackageJson.private;
         packageJson.scripts = loadedPackageJson.scripts;
-        packageJson.typings = loadedPackageJson.typings;
+        packageJson.typings = loadedPackageJson.typings || loadedPackageJson.types;
```

Comment:
I've always wondered about this. Are there docs that describe which is preferable?

Reply:
> Note that the "typings" field is synonymous with "types", and could be used as well.

The publishing docs seem to suggest that `types` is the original attribute, and that `typings` is an alternative alias. The example package.json files on that page all use `types`.